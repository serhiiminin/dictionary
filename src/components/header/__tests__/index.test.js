import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Header from '..';


describe('Header', () => {
  const component = renderer.create(
    <Router keyLength={0}>
      <Header/>
    </Router>
  );
  const tree = component.toJSON();

  test('render', () => {
    expect(tree).toMatchSnapshot();
  });
});
