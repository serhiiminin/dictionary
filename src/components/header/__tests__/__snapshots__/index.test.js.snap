// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header render 1`] = `
<div
  className={undefined}
>
  <h1>
    <a
      className="Header-headerLink-0-1-1"
      href="/"
      onClick={[Function]}
    >
      My dictionary
    </a>
  </h1>
</div>
`;
